From a2ef7f7d24112c40c094041e4413fedb06f17c37 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tom=C3=A1s=20Gonz=C3=A1lez?=
 <tomasagustin.gonzalezorlando@arm.com>
Date: Tue, 14 Mar 2023 11:16:38 +0000
Subject: [PATCH] net: zperf: fix typo in error messages

Fix typo in error messages from the zperf_tcp_receiver where UDP
is written instead of TCP.

Issue-Id: SCM-5823
Signed-off-by: Luca Fancellu <luca.fancellu@arm.com>
Upstream-Status: Backport [main:7c4ea08db8734eb616073640add51e9d11be8998]
---
 subsys/net/lib/zperf/zperf_tcp_receiver.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/subsys/net/lib/zperf/zperf_tcp_receiver.c b/subsys/net/lib/zperf/zperf_tcp_receiver.c
index 3cab08dc40a..3dc1f9bbc00 100644
--- a/subsys/net/lib/zperf/zperf_tcp_receiver.c
+++ b/subsys/net/lib/zperf/zperf_tcp_receiver.c
@@ -160,7 +160,7 @@ static void tcp_server_session(void)
 				 (struct sockaddr *)in4_addr_my,
 				 sizeof(struct sockaddr_in));
 		if (ret < 0) {
-			NET_ERR("Cannot bind IPv4 UDP port %d (%d)",
+			NET_ERR("Cannot bind IPv4 TCP port %d (%d)",
 				ntohs(in4_addr_my->sin_port), errno);
 			goto error;
 		}
@@ -216,7 +216,7 @@ static void tcp_server_session(void)
 				 (struct sockaddr *)in6_addr_my,
 				 sizeof(struct sockaddr_in6));
 		if (ret < 0) {
-			NET_ERR("Cannot bind IPv6 UDP port %d (%d)",
+			NET_ERR("Cannot bind IPv6 TCP port %d (%d)",
 				ntohs(in6_addr_my->sin6_port), errno);
 			goto error;
 		}
-- 
2.39.1

